$MirOS: ports/print/cups/patches/patch-config-scripts_cups-sharedlibs_m4,v 1.5 2006/11/08 18:25:44 bsiegert Exp $
--- config-scripts/cups-sharedlibs.m4.orig	Wed Dec  6 21:09:53 2006
+++ config-scripts/cups-sharedlibs.m4	Thu Mar 15 19:11:58 2007
@@ -61,7 +61,7 @@ if test x$enable_shared != xno; then
 			LIBCUPS="libcups.so.2"
 			LIBCUPSIMAGE="libcupsimage.so.2"
 			DSO="\$(CC)"
-			DSOFLAGS="$DSOFLAGS -Wl,-soname,\`basename \$@\` -shared \$(OPTIM)"
+			DSOFLAGS="$DSOFLAGS -shared \$(OPTIM)"
 			;;
 		Darwin*)
 			LIBCUPS="libcups.2.dylib"
